test/spec/modules/mwOpenLinkIdSystem_spec.js

```diff
@@ -0,0 +1,57 @@
+import { mwOpenLinkSubModule } from 'modules/mwOpenLinkIdSystem.js';
+import { newStorageManager } from 'src/storageManager.js';
+
+const storage = newStorageManager();
+
+const P_CONFIG_MOCK = {
+  params: {
+    accountId: '123',
+    partnerId: '123',
+    storage: true
```

Comment:
this isn't a valid value for `storage` in config, you should not be including it given your implementation is not using it.